Meadowlark-js/packages/meadowlark-core/src/handler/FrontendRequest.ts

```diff
@@ -32,11 +28,22 @@ export interface FrontendRequestMiddleware {
 
 export interface FrontendRequest {
   action: Action;
+  /**
+   * The URL path in the form /version/namespace/resource and optionally /resourceId
+   * The path should not include query parameters
+   */
   path: string;
+
+  /**
+   * A request identifier provided by the frontend service, used for log tracing
+   */
   traceId: string;
+
+  /**
+   * Unparsed request body provided by the frontend service as a string, or null if the is no body
+   */
   body: string | null;
   headers: FrontendHeaders;
-  pathParameters: FrontendPathParameters;
```

Comment:
Turns out this path parameterization stuff is something Meadowlark doesn't need.